src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -1559,4 +1560,20 @@ public void shouldGetDifferentTimestampLtzConsistentBetweenFormats() throws Exce
       }
     }
   }
+  // Test setHolderbility() API.
+  @Test
+  public void testHolderbility() throws Throwable {
```

Comment:
Typo - should be `testHoldability`. Also, no need to comment the method here as it is self-explanatory